aarch64_linux/build_aarch64_wheel.py

```diff
@@ -515,7 +519,7 @@ def start_build(host: RemoteHost, *,
         build_date = host.check_output("cd pytorch ; git log --pretty=format:%s -1").strip().split()[0].replace("-", "")
         version = host.check_output("cat pytorch/version.txt").strip()[:-2]
         build_vars += f"BUILD_TEST=0 PYTORCH_BUILD_VERSION={version}.dev{build_date} PYTORCH_BUILD_NUMBER=1"
-    if branch.startswith("v1."):
+    if branch.startswith("v"):
```

Comment:
Please be a bit more conservative here, as there are lots of branches that start with letter 'v', for example `viable/strict`:
```suggestion
    if branch.startswith("v1.") or branch.startswith("v2."):
```